City-of-Helsinki/parking-permits-ui
# Add Traficom copyright-markings to Webshop vehicle fields

## Description

This PR implements the following:
- Add Traficom-copyright next to each vehicle field.
- Remove Traficom-copyright from the Webshop-footer
- Update fi/sv/en-translations

## Context

[PV-659](https://helsinkisolutionoffice.atlassian.net/browse/PV-659)

## How Has This Been Tested?

Manually with each of the following views:
- Frontpage card
- Add vehicle page card
- Permit duration selector
- Change vehicle page card
- Change vehicle overview page

## Manual Testing Instructions for Reviewers

Check that the above pages are have fi/sv/en-localized Traficom-copyrights marked.
Also check that the Webshop footer does not have a generic Traficom-copyright anymore.

## Screenshots

<img width="632" alt="webshop - traficom vehicle details copyright fi" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/32f61c85-1ca9-4f61-ad85-4e20be0b97ff">

<img width="648" alt="webshop - traficom vehicle details copyright sv" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/e550ad2f-5545-4677-afc8-9dae69cfb734">

<img width="603" alt="webshop - traficom vehicle details copyright en" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/479126b2-4a53-4f8a-8b7b-f6a207d10c3a">

<img width="647" alt="webshop - traficom vehicle details copyright durationselector fi" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/54e43591-414c-4634-aee5-51a7f42a1f86">

<img width="611" alt="webshop - traficom vehicle details copyright durationselector sv" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/30849a9b-67b8-4a93-9293-5c97735e2fcc">

<img width="618" alt="webshop - traficom vehicle details copyright durationselector en" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/1ba74f4d-979a-41ac-9797-a5e5136ffb9f">

<img width="589" alt="webshop - frontpage fi" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/4c989509-ca4e-4877-a4df-6175dbf97277">

<img width="556" alt="webshop - frontpage sv" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/d8a009f8-4895-47a7-89e7-2399f7959a9b">

<img width="550" alt="webshop - frontpage en" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/7812fbce-9eae-422e-88a5-4eac5a36cf18">

<img width="548" alt="webshop - vehicle change fi" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/637a89cd-ac51-429e-a768-b712c1e39c80">

<img width="405" alt="webshop - vehicle change sv" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/f143cf2d-5bcd-43f7-8be1-ce8f895630b9">

<img width="385" alt="webshop - vehicle change en" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/6e0b6adc-97ed-4b6d-a8ff-5351f7d10bc5">

<img width="1148" alt="webshop - change vehicle overview page fi" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/669f2746-3c7c-45ac-a6ef-fb6f7bfdffb7">

<img width="1141" alt="webshop - change vehicle overview page sv" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/153b0722-e5c2-4b60-86aa-797f3684e794">

<img width="1143" alt="webshop - change vehicle overview page en" src="https://github.com/City-of-Helsinki/parking-permits-ui/assets/2784933/6bc0f326-11d3-4e91-92f3-1ffbca3858dd">
